api/src/Service/CampCouponService.php

```diff
@@ -57,4 +59,8 @@ public function isCouponFree($couponKey) {
 
         return 0 == $cnt;
     }
+
+    public function isActive(): bool {
```

Comment:
Shouldn't that be named `isDisabled`?

Or keep the name, but then the return value would need to be  `return '' !== $this->secret;`